SignalServiceKit/src/Messages/OWSDisappearingMessagesJob.m

```diff
@@ -366,12 +365,10 @@ - (void)timerDidFire
 {
     OWSAssert([NSThread isMainThread]);
 
-    // FIXME SHARINGEXTENSION
-    //    if ([UIApplication sharedApplication].applicationState != UIApplicationStateActive) {
-    //        // Don't run when inactive.
-    //        OWSFail(@"%@ Disappearing messages job timer fired while app inactive.", self.logTag);
-    //        return;
-    //    }
+    if (!CurrentAppContext().isMainAppAndActive) {
```

Comment:
Did you intend to lose this assert?, or do we want something more like:

```
if (CurrentAppContext().isMainApp) {
    OWSAssert(CurrentAppContext().isMainAppAndActive);
}
```

Reply:
Fixed.